src/containers/AudioUploader/components/AudioMetaData.jsx

```diff
@@ -27,7 +27,7 @@ const AudioMetaData = props => {
         description={t('form.tags.description')}>
         {({ field, form }) => (
           <AsyncSearchTags
-            locale={locale}
+            language={locale}
```

Comment:
locale er som regel nettleserspråket og ikkje valgt språk for elementet.

Reply:
så vi skal bruke locale som default bare hvis elementet ikke har språk?

Reply:
skal dette også gjelde concepts, bilder og artikler? at når man søker tags så gjør man det med elementets språk, og ikke locale?